nixos/lib/test-driver/test_driver/__init__.py

```diff
@@ -76,6 +76,11 @@ def main() -> None:
         nargs="*",
         help="vlans to span by the driver",
     )
+    arg_parser.add_argument(
+        "--junit-xml",
+        action="store_true",
+        help="Replace default logging with a junit-xml test report",
+    )
```

Comment:
What's your idea behind splitting the configuration options for this feature into an environment var and a command-line argument? With something like this, we could do it all on the command line:

```python
    parser.add_argument(
        "--junit-xml",
        required=False,
        help="Write a JUnit XML report to this file",
    )
```